frontend/app/src/components/home/upgrade/Features.svelte

```diff
@@ -57,6 +57,36 @@
         </div>
     </Feature>
 
+    <Feature>
```

Comment:
I would think don't show these features in the comparison table at all

Reply:
I think it's quite nice to have them just as a reference to what features exist and how lucky you all are to have access to them at no cost.

Reply:
By that reasoning we could add 10 more free features. I think they shouldn't be there.